test/mpi/CMakeLists.txt

```
@@ -1,4 +1,5 @@
 ginkgo_create_common_and_reference_test(assembly MPI_SIZE 3)
+ginkgo_create_common_and_reference_test(dd_matrix MPI_SIZE 3)
```

you can use this instead of the `#ifndef GKO_COMPILING_DPCPP` macro:
```suggestion
ginkgo_create_common_and_reference_test(dd_matrix MPI_SIZE 3 DISABLE_EXECUTORS dpcpp)
```